Clean up formatting errors

### Proposed Changes

```
netbox\dcim\base_filtersets.py:56:8: E121 continuation line under-indented for hanging indent
netbox\dcim\filtersets.py:1655:17: E126 continuation line over-indented for hanging indent
netbox\extras\tests\test_customfields.py:663:31: E203 whitespace before ','
netbox\ipam\models\vlans.py:364:13: E131 continuation line unaligned for hanging indent
netbox\ipam\models\vlans.py:372:13: E131 continuation line unaligned for hanging indent
```

### Justification

These errors were caught by pycodestyle:

pycodestyle --ignore=W504,E501 --exclude=node_modules netbox/

--- a/netbox/dcim/base_filtersets.py
+++ b/netbox/dcim/base_filtersets.py
@@ -53,10 +53,10 @@ class ScopedFilterSet(BaseFilterSet):
         label=_('Site (slug)'),
     )
     location_id = TreeNodeMultipleChoiceFilter(
-       queryset=Location.objects.all(),
-       field_name='_location',
-       lookup_expr='in',
-       label=_('Location (ID)'),
+        queryset=Location.objects.all(),
+        field_name='_location',
+        lookup_expr='in',
+        label=_('Location (ID)'),
     )
     location = TreeNodeMultipleChoiceFilter(
         queryset=Location.objects.all(),

--- a/netbox/dcim/filtersets.py
+++ b/netbox/dcim/filtersets.py
@@ -1652,8 +1652,8 @@ class MACAddressFilterSet(NetBoxModelFilterSet):
         if not value.strip():
             return queryset
         qs_filter = (
-                Q(mac_address__icontains=value) |
-                Q(description__icontains=value)
+            Q(mac_address__icontains=value) |
+            Q(description__icontains=value)
         )
         return queryset.filter(qs_filter)
 

--- a/netbox/extras/tests/test_customfields.py
+++ b/netbox/extras/tests/test_customfields.py
@@ -660,8 +660,8 @@ class CustomFieldAPITest(APITestCase):
             CustomField(
                 type=CustomFieldTypeChoices.TYPE_BOOLEAN,
                 name='boolean_field',
-                default=False)
-            ,
+                default=False
+            ),
             CustomField(
                 type=CustomFieldTypeChoices.TYPE_DATE,
                 name='date_field',

--- a/netbox/ipam/models/vlans.py
+++ b/netbox/ipam/models/vlans.py
@@ -361,7 +361,7 @@ class VLANTranslationRule(NetBoxModel):
     )
     local_vid = models.PositiveSmallIntegerField(
         verbose_name=_('Local VLAN ID'),
-            validators=(
+        validators=(
             MinValueValidator(VLAN_VID_MIN),
             MaxValueValidator(VLAN_VID_MAX)
         ),
@@ -369,7 +369,7 @@ class VLANTranslationRule(NetBoxModel):
     )
     remote_vid = models.PositiveSmallIntegerField(
         verbose_name=_('Remote VLAN ID'),
-            validators=(
+        validators=(
             MinValueValidator(VLAN_VID_MIN),
             MaxValueValidator(VLAN_VID_MAX)
         ),

--- a/netbox/virtualization/forms/model_forms.py
+++ b/netbox/virtualization/forms/model_forms.py
@@ -150,8 +150,8 @@ class ClusterAddDevicesForm(forms.Form):
                 for scope_field in ['site', 'location']:
                     device_scope = getattr(device, scope_field)
                     if (
-                        self.cluster.scope_type.model_class() == apps.get_model('dcim', scope_field)
-                            and device_scope != self.cluster.scope
+                        self.cluster.scope_type.model_class() == apps.get_model('dcim', scope_field) and
+                            device_scope != self.cluster.scope
                     ):
                         raise ValidationError({
                             'devices': _(

